README.md

```diff
@@ -1,6 +1,100 @@
-# Running
-- install postgresql (including the headers, `postgresql-server-dev` on ubuntu)
-- create a user and a database
-- run the following on your test database: `CREATE SEQUENCE users_usr_id_seq;`
-- import the users table
-- update the database and credentials in `config/database.yml`
+# PéK-Next
+
+**The administration system for Schönherz Zoltán Student Hostel and [SVIE](http://svie.hu/)**
+
+[![Build Status](https://travis-ci.org/kir-dev/pek-next.png?branch=master)](https://travis-ci.org/kir-dev/pek-next)
+[![Code Climate](https://codeclimate.com/github/kir-dev/pek-next.png)](https://codeclimate.com/github/kir-dev/pek-next)
+[![Dependency Status](https://gemnasium.com/kir-dev/pek-next.png)](https://gemnasium.com/kir-dev/pek-next)
+
+## Running
+
+**To run PéK you will need the following:**
+
+###### Postgresql packages
+
+```bash
+$ sudo apt-get install postgresql libpq-dev nodejs
+```
+
+###### Ruby 2.2.3 environment
+
+```bash
+$ \curl -sSL https://get.rvm.io | bash -s stable
+$ rvm install 2.2.3
```

Comment:
what do you think about using the latest one (2.4.2)? should be seamless upgrade

Reply:
👍 I also think it's worth upgrading to 2.4.2